Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags-vpost-7.4.md

```diff
@@ -80,3 +80,38 @@ Notice that Newtonsoft.Json is referenced in the below example. It's already a p
     </ul>
 }
 ```
+### Setting Tags Programatically:
+You can use the ContentService to create and update Umbraco content from c# code, when setting tags there is an extension method (SetTags) on IContentBase that helps you set the value for a Tags properties. Remember to add the using statement for `Umbraco.Core.Models` to take advantage of it.
```

Comment:
```suggestion
You can use the ContentService to create and update Umbraco content from c# code, when setting tags there is an extension method (SetTags) on IContentBase that helps you set the value for a Tags property. Remember to add the using statement for `Umbraco.Core.Models` to take advantage of it.
```

Reply:
Yes, it should be in the other two, I was editing online so I'm not sure how it editing this version - the extension method has been there for always - it's been renamed in V8 as AssignTags, if we can add that to a list somewhere!